streamingpro-it/src/test/scala/tech/mlsql/it/LocalBaseTestSuite.scala

```diff
@@ -51,7 +51,8 @@ trait LocalBaseTestSuite extends FunSuite with SparkOperationUtil with BeforeAnd
     setupRunParams()
     copyDataToUserHome(user)
     TestManager.loadTestCase(new File(testCaseDirPath))
-    PlatformManager.getOrCreate.run(new ParamsUtil(runParams))
+    // Load built-in and external plug-ins and start PlatformManager
```

Comment:
This is a bug in streamingpro-it ？ If so ,please issue a new PR instead in this PR .